keycloak/themes/uid2-theme/login/template.ftl

```diff
@@ -45,7 +45,8 @@
     <div class="centralize-content">
     <#-- App-initiated actions should not see warning messages about the need to complete the action -->
     <#-- during login.                                                                               -->
-    <#if displayMessage && message?has_content && (message.type != 'warning' || !isAppInitiatedAction??)>
+    <br />
+    <#if displayMessage && message?has_content && message.type != 'warning' && !isAppInitiatedAction??>
```

Comment:
just taking out the parentheses fixed the logic here and no longer shows the "activate your account" message but still shows the error messages if someone inputs the wrong password